lib/tapjoy/autoscaling_bootstrap/elb.rb

```diff
@@ -2,16 +2,30 @@ module Tapjoy
   module AutoscalingBootstrap
     # This class configures elastic load balancers
     class ELB
-      # Initialize the class
-      def initialize
+      def initialize(elb_hash, clobber_elb, zones, security_groups)
+        elb_config = build_config(elb_hash, zones, security_groups)
+        check_valid_config(elb_config)
+
+        if exists && clobber_elb
+          delete
+        elsif exists
+          return
```

Comment:
oops.  This was a placeholder.  I'll have a new update that should fix this and answer your previous question.